justfile

```diff
@@ -14,8 +14,6 @@ default: mise mvn
 
 # mise install
 mise:
-    mise plugin install --quiet maven
```

Comment:
`maven` has moved [into the list of mise core plugins](https://github.com/jdx/mise/blob/v2025.1.15/registry.toml#L1193-L1196).